eng/build.yml

```
@@ -136,6 +136,7 @@ jobs:
         condition: succeeded()
 
     - ${{ if eq(parameters.osGroup, 'MacOS') }}:
+      - script: /bin/bash -c "sudo xcode-select -s /Applications/Xcode_10.3.app/Contents/Developer"
```

Please condition this only to happen on public (the only one that uses that queue)
```suggestion
      - ${{ if eq(variables['System.TeamProject'], 'public') }}:
        - script: /bin/bash -c "sudo xcode-select -s /Applications/Xcode_10.3.app/Contents/Developer"
```